erpnext/agriculture/doctype/plant_batch/plant_batch.py

```diff
@@ -105,6 +99,16 @@ def get_coordinates(doc):
 def get_geometry_type(doc):
 	return ast.literal_eval(doc.location).get('features')[0].get('geometry').get('type')
 
+def validate_quantities(self, destroy_count):
+		if self.untracked_count == 0:
+			frappe.throw(_("The plant batch must have an untracked count."))
+
+		if int(destroy_count) <= 0 :
+			frappe.throw(_("Destroy count cannot be less than or equal to 0."))
+
+		if self.untracked_count < int(destroy_count):
+			frappe.throw(_("The Destroy Count ({0}) should be less than or equal to the untracked count ({1})").format(destroy_count,self.untracked_count))
```

Comment:
- There is no need to move this to different function since it is called only in `validate_plant_batch_quantities`.
- Also if you want to define a function outside of the class, refrain from using `self`, it'll work no doubt, but `self` is generally defined inside of class.